DistroLauncher/ProcessRunner.cpp

```diff
@@ -1,3 +1,22 @@
+//
+//    Copyright: 2021, Canonical Ltd.
+//  License: GPL-3
+//  This program is free software : you can redistribute itand /or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License.
```

Comment:
Are you sure this is the standard GPL3 header? (the ", either version of 3" reads weirdly…)

Some snippets from Unity for instance is:
```
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License version 3 as
 * published by the Free Software Foundation.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.

```

Reply:
Well, I should have asked instead... Before I started to add our functions in separate files not present in the upstream, the only file with a Canonical copyright in this repository was the python script `lp-distro-info`, which is copyrighted as GPL 3. So I just did Ctrl-C Ctrl-V 😅 .

Also, the upstream repository is MIT licensed.

What do you recommend for that?

Reply:
GPL-3 and blame @jibel for the incorrect template :p

Reply:
🤣 
So, copy-paste from Unity and make sure that python script will get fixed as well.